src/components/pages/Users/EditUserPanel/EditUserPanel.tsx

```diff
@@ -241,13 +273,30 @@ const EditUserPanel = ({ close, user, userId, setPanelWidth }: Props) => {
               </ToastCard>
             ));
           }
-          if (!hasGroupsError && !hasRolesError && !hasEntitlementsError) {
+          if (hasUserError) {
+            reactHotToast.custom((t) => (
+              <ToastCard toastInstance={t} type="negative">
+                {Label.USER_ERROR}
+              </ToastCard>
+            ));
+          }
+          if (
+            !hasGroupsError &&
+            !hasRolesError &&
+            !hasEntitlementsError &&
+            !hasUserError
+          ) {
             reactHotToast.custom((t) => (
               <ToastCard toastInstance={t} type="positive">
                 {`User with email "${user.email}" was updated.`}
               </ToastCard>
             ));
           }
+          if (userQueryKey && !hasUserError) {
+            void queryClient.invalidateQueries({
+              queryKey: userQueryKey,
+            });
+          }
```

Comment:
I went with a slightly different approach where I provided a prop that was called if the entity was updated: https://github.com/canonical/rebac-admin/pull/180/files#diff-670678abec75c96807b95b639999f711107552723a6231b735b477996eca96d7R75.

Not sure which method is better...

Reply:
I've changed the code to utilize your method in order for the codebase to remain consistent.